external-crates/move/crates/move-cli/src/base/new.rs

```diff
@@ -58,27 +58,60 @@ impl New {
         create_dir_all(path.join(SourcePackageLayout::Sources.path()))?;
         let mut w = std::fs::File::create(path.join(SourcePackageLayout::Manifest.path()))?;
         writeln!(
-            &mut w,
+            w,
             "[package]
 name = \"{name}\"
 
+# version = \"0.1.1\"  # semver \"maj.min.patch\"
```

Comment:
FYI we can (should) discourage `version` as a field since it wasn't effectively used and made optional in https://github.com/MystenLabs/sui/pull/15148 slated for removal. (CC @amnn)

I tried nuking it from existing `Move.toml` in https://github.com/MystenLabs/sui/pull/15463/files but we need to (at least) still [fix TypeScript parsing these] which I'll get around to https://github.com/MystenLabs/sui/actions/runs/7294939629/job/19880618043?pr=15463